src/pages/migrating/benefits.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Benefits
+description:
+  Carbon is for all members of a cross-functional product team—designers,
+  developers, marketers and offering managers. Every point along a user's
+  journey needs to feel part of a holistic, branded experience—that's where
+  Carbon comes in.
+---
+
+<PageDescription>
+  What are the benefits of migrating to v11 as a designer, developer, product
+  manager or as my product's customers?
+</PageDescription>
+
+## As a designer
+
+- Token names have been updated for easier application. Names are more logical
+  and easier to apply without always having to refer to a guide.
+- Nothing visually has changed in any of the components, and no redesign work is
+  required.
+
+## As a developer
+
+- CSS grid provides easier implementation of layouts, support for sub-grid, and
+  other features that come along with this new technology.
+- CSS custom properties for theming (light mode / dark mode).
+- A 90% reduction in compile times for most projects with the update to Sass
```

Comment:
The 90% reduction time percent may be used too freely here. We noticed this in our work but might be more accurate to say that we reduced sass compilation times generally.